edgetxInitPassthru.py

```diff
@@ -28,9 +28,12 @@ def find_radio_serial_ports():
         return None
     radioportList = []
     for port in portList:
+        print('*',port.device, port.name, port.description)
```

Comment:
can you pl take them out again, or at least comment out
(I took them out since I figured in order to test what one sees one can/should use the general find serial function)

Reply:
Yes, sorry, I got interrupted for a couple of days and thought I had removed this already.